joomla/joomla-cms
# Fixing the testing sample data for newsfeeds

Pull Request for Issue #13979 .

### Summary of Changes

The testing sample data points to a non existing neewsfeed id `4` this is changed to id `1`
The testing sample data has an invalid RSS feed for the module this have been moved to `http://feeds.joomla.org/JoomlaMagazine`

Thanks @infograf768  for the analyse and @franz-wohlkoenig for finding the issues.

### Testing Instructions

Install: https://github.com/zero-24/joomla-cms/archive/installation_fixes.zip
Confirm the `Single News Feed` menu entry works now
Confirm the  `Feed Display` works now too

### Expected result

The `Single News Feed` menu entry work
The  `Feed Display` works

### Actual result

The `Single News Feed` menu entry does not work
The  `Feed Display` does not work

### Documentation Changes Required

None.
